tarantool/ext_types/packer.py

```diff
@@ -0,0 +1,9 @@
+from decimal import Decimal
+from msgpack import ExtType
+
+import tarantool.ext_types.decimal as ext_decimal
+
+def default(obj):
+    if isinstance(obj, Decimal):
+        return ExtType(ext_decimal.EXT_ID, ext_decimal.encode(obj))
+    raise TypeError("Unknown type: %r" % (obj,))
```

Comment:
Yeah, there is a separate issue for it: https://github.com/tarantool/tarantool-python/issues/191

Maybe it will be solved in #228 since this is where some controversy may raise. Actually it's not a big deal -- we just need to provide a couple of function argument to Connection object with out extended types packer/unpacker as a default.